jetty-ee9/jetty-ee9-webapp/src/main/java/org/eclipse/jetty/ee9/webapp/WebAppContext.java

```diff
@@ -746,13 +740,13 @@ public String[] getServerClasses()
     }
 
     @Override
-    public boolean isServerClass(Class<?> clazz)
+    public boolean isHiddenClass(Class<?> clazz)
     {
         return _serverClasses.match(clazz);
     }
 
     @Override
-    public boolean isSystemClass(Class<?> clazz)
+    public boolean isProtectedClass(Class<?> clazz)
     {
         return _systemClasses.match(clazz);
```

Comment:
Still calling this  _systemClasses and not _protectedClasses (like you did in ee10?)

Reply:
I didn't want to change too much in the ee9 implementation